src/components/Search/SearchViewInit.tsx

```diff
@@ -1,16 +1,17 @@
 import { useEffect } from 'react';
 
-import { connect, useDispatch } from 'react-redux';
+import { connect } from 'react-redux';
 
+import { useAppDispatch } from '../../hooks/app';
 import { fetchRecentRevisions } from '../../thunks/searchThunk';
-import { Repository, State } from '../../types/state';
+import type { Repository, State } from '../../types/state';
 
 // component to fetch recent revisions when search view is loaded
 function SearchViewInit(props: SearchViewInitProps) {
-  const dispatch = useDispatch();
+  const dispatch = useAppDispatch();
   const { repository } = props;
   useEffect(() => {
-    dispatch(fetchRecentRevisions(repository));
+    void dispatch(fetchRecentRevisions(repository));
```

Comment:
`Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator.`

this promise is awaited in the thunk action